pkg/agent/controller/networkpolicy/audit_logging_test.go

```diff
@@ -110,7 +111,7 @@ func TestDropPacketDedupLog(t *testing.T) {
 }
 
 func TestDropPacketMultiDedupLog(t *testing.T) {
-	antreaLogger, mockAnpLogger := newTestAntreaPolicyLogger()
+	antreaLogger, mockAnpLogger := newTestAntreaPolicyLogger(100 * time.Millisecond)
 	ob, expected := newLogInfo("Drop")
 
 	go sendMultiplePackets(antreaLogger, ob, 10)
```

Comment:
I think @tnqn had a concern about the stability of this test. Maybe for this test we can send a packet every 40ms so that there is a bigger margin of error? We can also use a virtual clock for the tests that we advance manually, but that can be a bit tedious to do.

Reply:
I tried running the test 100 times locally with enough CPU resources and it didn't fail a single time.
I tried limitting CPU to 30% then it failed 3/100 times.
```
# cgcreate -g cpu:/cpulimited
# cgset -r cpu.cfs_quota_us=30000 cpulimited
# cgexec -g cpu:cpulimited go test -race -count 100 -run TestDropPacketMultiDedupLog ./pkg/agent/controller/networkpolicy/
--- FAIL: TestDropPacketMultiDedupLog (0.22s)
    audit_logging_test.go:130:
                Error Trace:    audit_logging_test.go:130
                Error:          "2021/08/12 AntreaPolicyIngressRule AntreaNetworkPolicy:default/test Drop 0 SRC: 0.0.0.0 DEST: 1.1.1.1 60 TCP [8 packets in 100.404692ms]\n" does not contain "AntreaPolicyIngressRule AntreaNetworkPolicy:default/test Drop 0 SRC: 0.0.0.0 DEST: 1.1.1.1 60 TCP [9"
                Test:           TestDropPacketMultiDedupLog
--- FAIL: TestDropPacketMultiDedupLog (0.22s)
    audit_logging_test.go:130:
                Error Trace:    audit_logging_test.go:130
                Error:          "2021/08/12 AntreaPolicyIngressRule AntreaNetworkPolicy:default/test Drop 0 SRC: 0.0.0.0 DEST: 1.1.1.1 60 TCP [6 packets in 100.735445ms]\n" does not contain "AntreaPolicyIngressRule AntreaNetworkPolicy:default/test Drop 0 SRC: 0.0.0.0 DEST: 1.1.1.1 60 TCP [9"
                Test:           TestDropPacketMultiDedupLog
--- FAIL: TestDropPacketMultiDedupLog (0.22s)
    audit_logging_test.go:130:
                Error Trace:    audit_logging_test.go:130
                Error:          "2021/08/12 AntreaPolicyIngressRule AntreaNetworkPolicy:default/test Drop 0 SRC: 0.0.0.0 DEST: 1.1.1.1 60 TCP [8 packets in 100.463209ms]\n" does not contain "AntreaPolicyIngressRule AntreaNetworkPolicy:default/test Drop 0 SRC: 0.0.0.0 DEST: 1.1.1.1 60 TCP [9"
                Test:           TestDropPacketMultiDedupLog
```
I'm not sure how much CPU resources github action can provide. But I'm fine with keeping it as is and fixing it when we see actual failure.

Reply:
Found a test failure in another PR: https://github.com/antrea-io/antrea/pull/2579/checks?check_run_id=3308830375, so Github action doesn't provide enough CPU resources and the test needs to be adjusted.
```
--- FAIL: TestDropPacketMultiDedupLog (0.22s)
    audit_logging_test.go:118: 
        	Error Trace:	audit_logging_test.go:118
        	Error:      	"2021/08/12 AntreaPolicyIngressRule AntreaNetworkPolicy:default/test Drop 0 SRC: 0.0.0.0 DEST: 1.1.1.1 60 TCP [6 packets in 100.0035ms]\n" does not contain "AntreaPolicyIngressRule AntreaNetworkPolicy:default/test Drop 0 SRC: 0.0.0.0 DEST: 1.1.1.1 60 TCP [9"
```

Reply:
Just found this PR's own windows unit test failed too, so it's quite flaky on windows instances.

Reply:
Since the log result depends on CPU resources, does it make sense to alter the testing for `MultiDedupLog` a little? Testing that there are **at least two** log messages, and the **sum** of these msg equal to requested? Instead of non-determinant msg distribution. Thanks!

Reply:
that makes sense to me, thanks @qiyueyao
it's still sensitive to the clock, but we can ensure that there is a comfortable margin

Reply:
Thanks, I used the suggested solution and tested with 40% cpu power & 100 counts, it works fine.